core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -279,19 +291,22 @@ public void init() {
         }
       } catch (RuntimeException e) {
         while (!initiatedCaches.isEmpty()) {
+          String toBeClosed = initiatedCaches.pop();
           try {
-            removeCache(initiatedCaches.pop());
+            removeCache(toBeClosed);
           } catch (Exception exceptionClosingCache) {
-            // todo probably should log these exceptions
+              LOGGER.error("Cache '"+toBeClosed+"' could not be removed due to "+ exceptionClosingCache.getMessage());
```

Comment:
When logging an exception, pass also the exception as param to the method, so that the stack trace is available in the log file.